Modules/_sqlite/prepare_protocol.c

```diff
@@ -46,10 +46,13 @@ pysqlite_prepare_protocol_dealloc(pysqlite_PrepareProtocol *self)
     Py_DECREF(tp);
 }
 
+PyDoc_STRVAR(doc, "PEP-246 style object adaption protocol type.");
```

Comment:
```suggestion
PyDoc_STRVAR(doc, "PEP 246 style object adaption protocol type.");
```

Canonical spelling of PEP